python/examples/sklearn_vw.ipynb

```diff
@@ -2,24 +2,19 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {
-    "collapsed": false
-   },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "training score: 0.522125\n",
-      "testing score: 0.511\n"
-     ]
-    }
-   ],
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
    "source": [
     "# Binary Classification\n",
+    "\n",
     "from __future__ import print_function\n",
     "\n",
+    "# import warnings filter\n",
+    "from warnings import simplefilter\n",
+    "# ignore all future warnings\n",
+    "simplefilter(action='ignore', category=FutureWarning)\n",
```

Comment:
Which future warnings are being ignored by this?

Reply:
Currently, we are importing LinearClassifierMixin and SparseCoefMixin from sklearn.linear_model.base. In version v0.24 this importing will be removed and currently it gives the deprecation warning. I thought that in example, when someone tries they should not be concerned with the warning but just the output to learn. I'm currently working on the fix for the issue as well.

Reply:
@jackgerrits If you are satisfied with my response, can you please mark the comment as resolved?

Reply:
Can you please add a comment explaining this?

Reply:
Done.